data/lang-en.csv

```diff
@@ -47,43 +47,51 @@ Dashboard.Running.Running,Running:,
 Dashboard.Running.EstimatedTimeLeft,Estimated time left:,
 Dashboard.Running.Seconds,{{seconds}} seconds,
 Dashboard.Running.PreparingTest,Preparing test,
+Dashboard.Running.CalculatingETA,Calculating ETA,
 Dashboard.Running.ShowLog,Show Log,
 Dashboard.Running.CloseLog,Close Log,
 Dashboard.Card.Subtitle,Tap card for more,
 Dashboard.Card.Seconds,~{{seconds}}s,
 Dashboard.Websites.Card.Description,Test the blocking of websites,
-Dashboard.Websites.Overview.Paragraph,"Check whether websites are blocked using OONI's [Web Connectivity test](https://ooni.io/nettest/web-connectivity/).
-
-Every time you tap Run, you test different websites from the Citizen Lab's [global](https://github.com/citizenlab/test-lists/blob/master/lists/global.csv) and [country-specific](https://github.com/citizenlab/test-lists/tree/master/lists) test lists.
-
-To test the sites of your choice, tap the Choose websites button or select categories of sites via the settings of this card. 
-
-This test measures whether websites are blocked by means of DNS tampering, TCP/IP blocking or by a transparent HTTP proxy.
-
-Your results will be published on [OONI Explorer](https://explorer.ooni.io/world/) and [OONI API](https://api.ooni.io/).",
+Dashboard.Websites.Overview.Paragraph,"Check whether websites are blocked using OONI's [Web Connectivity test](https://ooni.io/nettest/web-connectivity/).
+
+Every time you tap Run, you test different websites from the Citizen Lab's [global](https://github.com/citizenlab/test-lists/blob/master/lists/global.csv) and [country-specific](https://github.com/citizenlab/test-lists/tree/master/lists) test lists.
+
+To test the sites of your choice, tap the Choose websites button or select categories of sites via the settings of this card. 
+
+This test measures whether websites are blocked by means of DNS tampering, TCP/IP blocking or by a transparent HTTP proxy.
+
+Your results will be published on [OONI Explorer](https://explorer.ooni.org/) and [OONI API](https://api.ooni.io/).",

```

Comment:
Perhaps rows 56-64 need to be deleted (as this is the copy for the Websites card in the mobile app)?

Reply:
For now, it is easier to leave it there since we still share the spreadsheet. Otherwise, I will have to do such deletions manually, every time I update the copy. Lets find a suitable solution for this separately.